src/sql/parts/profiler/service/profilerService.ts

```diff
@@ -82,6 +82,11 @@ export class ProfilerService implements IProfilerService {
 		this._sessionMap.get(this._idMap.reverseGet(params.sessionId)).onMoreRows(params);
 	}
 
+	public onSessionStopped(params: sqlops.ProfilerSessionStoppedParams): void {
+
+		this._sessionMap.get(this._idMap.reverseGet(params.ownerUri)).onSessionStopped(params);
```

Comment:
do we want to remove the session from the `_sessionMap` when it's stopped?  Or do we still need to be able to access it after the stop notification?

Reply:
This session is actually a bit misleading, it's not referring to the session on the database, only to the open profiler session window. In my head I refer to it as the profiler tab session, or viewer session. Other than having it's state changed, it's unaffected by starting or stopping the XEvent session, we'd only remove it if the window was closed.